api/v1beta1/placementapi_types.go

```diff
@@ -86,6 +86,11 @@ type PlacementAPISpec struct {
 	// actual action pod gets started with sleep infinity
 	Debug PlacementAPIDebug `json:"debug,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// PlacementAPIImmutable - make some specs immutable to prohibit update operations
```

Comment:
Please describe the logic at the DatabaseInstance field too.

Reply:
I've added the description of that field here: https://github.com/openstack-k8s-operators/placement-operator/pull/17/files#diff-e5c063113f25a841c51f8370d6711ece1751aa814b24f925f440881290cb6e59R170

Or do you intend to adding any additional description to this immutable key level ?

Reply:
This is no longer applicable in the latest version.